### PR TITLE
Added a way to select the age limit

### LINKED ISSUE #12: Select the age limit
Provide a way to be able to select 3 age groups
Over 18,
Over 21 default
Over 60

With a user interface as follow:
Green Circle with "18+"
Red Circle with " 18-"

Green Circle with "21+"
Red Circle with " 22-"

Green Circle with "60+"
Red Circle with " 60-"

This could be a simple selection on the home screen, or have its own settings page.

### PR DESCRIPTION
**Pull request by Chrishon-Socket**
closes #12

### CODE REVIEW

File: Over21/ViewController.swift [hunk 1]
```diff
@@ -56,12 +72,19 @@ class ViewController: UIViewController {
...
         ageIndicatorView.leadingAnchor.constraint(equalTo: view.leadingAnchor).isActive = true
         ageIndicatorView.topAnchor.constraint(equalTo: view.topAnchor).isActive = true
         ageIndicatorView.trailingAnchor.constraint(equalTo: view.trailingAnchor).isActive = true
         ageIndicatorView.bottomAnchor.constraint(equalTo: view.bottomAnchor).isActive = true
         
         
+        ageLimitSelectionView.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        ageLimitSelectionView.topAnchor.constraint(equalTo: ageIndicatorView.scannerConnectionLabel.bottomAnchor, constant: 12).isActive = true
+        ageLimitSelectionView.widthAnchor.constraint(equalToConstant: 160.0).isActive = true
+        //ageLimitSelectionView.heightAnchor.constraint(equalToConstant: view.bottomAnchor).isActive = true
```
Comment (EricGlaenzer) [inline Over21/ViewController.swift]: should we remove this commented out code?

Comment (Chrishon-Socket) [reply Over21/ViewController.swift]: Removed in latest commit.

Comment (EricGlaenzer) [reply Over21/ViewController.swift]: Did you push it because I don't see it removed?

Comment (Chrishon-Socket) [reply Over21/ViewController.swift]: Yea, I pushed this change 4 minutes ago.

**Pull request merged**